### PR TITLE
Remove smallButton from button.less

### LINKED ISSUE #9232: Remove smallButton
**Describe the issue you encountered:** Remove `smallButton` from button.less. Since (at least) a half year the classname has only been used on [findbar.js](https://github.com/brave/browser-laptop/blob/64c48d5039b5ab66c45b5fdd5be68206ffd6aa89/app/renderer/components/main/findbar.js#L241). Refactoring findbar with Aphrodite, the classname can be removed from button.less.

Also please refer the discussion here: https://github.com/brave/browser-laptop/commit/615a6ec152a4535531a5cf46b7d6489d73060d62#commitcomment-20830663

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
The classname has only been used on findbar.

Closes #9232
Addresses #9233

Auditors:

Test Plan:
1. npm run test -- --grep='findBar'
2. Open the browser
3. Display find bar
4. Run manual test plan on https://github.com/brave/browser-laptop/wiki/Manual-Tests#find-on-page

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: js/about/styles.js [hunk 1]
```diff
@@ -261,13 +261,6 @@ class AboutStyle extends ImmutableComponent {
...
-        {/* TODO: This button size doesn't match its name */}
-        <button data-l10n-id='smallButton' className='browserButton whiteButton smallButton' onClick={this.onRemoveBookmark} />
-        <Pre><Code>
-          &lt;button data-l10n-id='done' className='browserButton whiteButton smallButton'{'\n'}
-          onClick={'{this.onRemoveBookmark}'} />
-        </Code></Pre>
-
         <BrowserButton primaryColor l10nId='primaryColor' onClick={this.onRemoveBookmark} />
         <Pre><Code>
           &lt;BrowserButton l10nId='cancel' onClick={'{this.onRemoveBookmark}'} />
```
Comment (luixxiul) [inline js/about/styles.js]: the bug here (it lacks `primaryColor`) is going to be fixed with https://github.com/brave/browser-laptop/pull/9239/files#diff-4f8a61660fd4d0a235614f82308c769eR273

**Comment by luixxiul** [comment]
@cezaraugusto would you mind adding iconOnly to style.js?

**Comment by cezaraugusto** [comment]
I took advantage of this removal and included `iconOnly` button, which may benefit downloadBar icons and maybe other icon-only buttons, lmk your thoughts and feel free to tweak

**Comment by cezaraugusto** [comment]
@luixxiul rebased after 40799f92cd0f32eec7b0764963eb41ce655b7ee9 pls check again. Added some fancy props to iconOnly so we could apply for all icon-buttons we have.

**Pull request merged**